DGTCentaurMods/opt/DGTCentaurMods/game/menu.py

```diff
@@ -230,6 +233,26 @@ def show_welcome():
 show_welcome()
 epaper.quickClear()
 
+
+def get_lichess_client():
+    logging.debug("get_lichess_client")
+    import berserk
+    import berserk.exceptions
+    token = centaur.get_lichess_api()
+    if not len(token):
+        logging.error('lichess token not defined')
+        raise ValueError('lichess token not defined')
+    session = berserk.TokenSession(token)
+    client = berserk.Client(session=session)
+    # just to test if the token is a valid one
+    try:
+        who = client.account.get()
+    except berserk.exceptions.ResponseError:
+        logging.error('Lichess API error. Wrong token maybe?')
+        raise
+    return client
+
+
```

Comment:
I would put this function somewhere out of the menu.py, like inside a module containing lichess helper functions in `/board` so that we don't get lost looking where a function is defined. note: all modules in `/board` are loaded by menu.py at start.